### DISCUSSION TITLE
inserire in HA una pagina http, come?

### DISCUSSION THREAD

**Discussion by recluta76** [indomus/forum#2392] [HUB: Home Assistant]
ciao, non so se sia fattibile, ho provato a cercare ma non ho trovato molto.
su un indirizzo ip interno 192.168.1.xx ho l'allarme. Accedendo a questo indirizzo, si apre la pagina dell'allarme con tutte le relative attività.
E' possibile inserire questo http all'interno di HA? vorrei creare nella plancia una pagina dedicata a questo indirizzo, in modo da avere in un'unica schermata tutto l'allarme.
Sarebbe possibile?
grazie

**Comment by snakuzzo** [comment]
Certo che puoi con [panel iframe](https://www.home-assistant.io/integrations/panel_iframe/), però se hai HA esposto in https, ovviamente non potrai aprire pagine http. Se lo raggiungi dalla rete in http interna invece sì.

**Reply by recluta76** [reply]
Ciao Snakuzzo, grazie per la risposta. ho HA esposto in https. Dal pannello, quindi, interna alla rete avrò la pagina dell'allarme e se mi dovessi collegare da fuori, no.. grazie

**Comment by recluta76** [comment]
ho messo il pannello su HA ma quando clicco sul pannello, mi appare la pagina iniziale dell'allarme con l'inserimento del codice ma mi appare questo messaggio: 
attenzione per effettuare l'accesso al sistema è necessario che il browser supporti i cookie e che l'opzione di attivazione dei cookie sia selezionata

uso chrome e l'opzione di accettazione dei cookies è attiva. (ho attivato accetta tutti i cookie).

dove sbaglio?grazie

**Reply by snakuzzo** [reply]
Mi sa che non puoi farci nulla.

**Reply by recluta76** [reply]
è qualcosa legato, però, ad HA perché accedendo alla pagina da browser non mi appare quel messaggio...

**Reply by snakuzzo** [reply]
Non è legato nello specifico ad HA... è un discorso un tantinello più complesso...

https://medium.com/trabe/cookies-and-iframes-f7cca58b3b9e

**Reply by recluta76** [reply]
hai ragione...e chi lo avrebbe mai detto...

**Reply by recluta76** [reply]
Ho risolto! in base all'articolo che mi hai postato, ho cercato un browser un pò attento a quella vicenda ed ho provato con firefox e va senza problemi! grazie ancora!
